client/http/client.go

```diff
@@ -17,31 +19,53 @@ type HttpClientInterface interface {
 }
 
 type HttpClient struct {
-	ApiKey    string
-	ApiSecret string
-	Endpoint  string
-	client    *resty.Client
+	ApiKey      string
+	ApiSecret   string
+	Endpoint    string
+	client      *resty.Client
+	rateLimiter *rate.Limiter
 }
 
 type HttpClientConfig struct {
-	ApiKey      string
-	ApiSecret   string
-	ApiEndpoint string
-	UserAgent   string
-	RestClient  *resty.Client
+	ApiKey                  string
+	ApiSecret               string
+	ApiEndpoint             string
+	UserAgent               string
+	RestClient              *resty.Client
+	RateLimitPerMinute      int // Optional, defaults to 500 if not specified
+	RateLimitAccumulateRate int // Optional, defaults to 8 if not specified
 }
 
 func NewHttpClient(config HttpClientConfig) (*HttpClient, error) {
+	rateLimitPerMinute := config.RateLimitPerMinute
```

Comment:
if we are not planning to change that, why not just hardcode the default values right now?